runtime/phala/build.rs

```diff
@@ -15,12 +15,13 @@
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
 fn main() {
-    #[cfg(feature = "std")]
+    #[cfg(all(feature = "std", feature = "metadata-hash"))]
     {
         substrate_wasm_builder::WasmBuilder::new()
             .with_current_project()
             .export_heap_base()
             .import_memory()
+            .enable_metadata_hash("UNIT", 12)
```

Comment:
Didn't check the parameters, but I guess it should be ("PHA", 12)?

Reply:
Ok i'll fix it. By the way, the github pipeline use rust 1.73, but as i mentioned, now we must use rust 1.75, you may upgrade it on the pipline server.